sourcemash/items/models.py

```diff
@@ -13,6 +13,7 @@ class Item(db.Model):
     category_1 = db.Column(db.String(100))
     category_2 = db.Column(db.String(100))
     voteSum = db.Column(db.Integer, default=0)
+    trending = db.Column(db.Integer, default=0) # total number of votes cast
```

Comment:
Do we have an API call to pull out the top X trending items? Like /api/items/trending?